modules/observability/metrics-server/base/resources/rbac/clusterroles.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Minor thing, based on how the aggregation works we can leave only the `aggregate-to-view` label and the others will pick it up the same
```suggestion
    rbac.authorization.k8s.io/aggregate-to-view: "true"
```